### PR TITLE
add dedalus

### PR DESCRIPTION
**Pull request by rluria14**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by rluria14** [comment]
@oblute @ndmaxar agree to be maintainers?

File: recipes/dedalus/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  number: 0
+  skip: True  # [win or py>37]
+  script:
+    - export FFTW_PATH=$PREFIX
+    - export FFTW_INCLUDE_PATH=$PREFIX/include
+    - export FFTW_LIBRARY_PATH=$PREFIX/lib
+    - export MPI_PATH=$PREFIX
+    - export MPI_INCLUDE_PATH=$PREFIX/include
+    - export MPI_LIBRARY_PATH=$PREFIX/lib
+    - {{ PYTHON }} -m pip install . -vv
```
Comment (scopatz) [inline recipes/dedalus/meta.yaml:24]: This should probably be in a build.sh script

File: recipes/dedalus/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - pip
+    - python
+    - setuptools
+    - cython
+    - mpi4py
+    - numpy
```
Comment (scopatz) [inline recipes/dedalus/meta.yaml:35]: See https://conda-forge.org/docs/maintainer/knowledge_base.html#building-against-numpy

File: recipes/dedalus/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    - fftw
+    - fftw * {{ mpi_prefix }}_*
+
+  run:
+    - python
+    - docopt
+    - h5py
+    - fftw
+    - fftw * {{ mpi_prefix }}_*
+    - matplotlib
```
Comment (scopatz) [inline recipes/dedalus/meta.yaml:45]: Should be `matplotlib-base`

Comment (rluria14) [reply recipes/dedalus/meta.yaml]: @scopatz these changes have been made. thanks!

**Comment by rluria14** [comment]
@scopatz @conda-forge/staged-recipes ready for final review and merge

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**